index.html

```diff
@@ -0,0 +1,112 @@
+      <div id="lists"><img src="assets/icons/note-icon.svg" alt="">Lists</div>
+      <div id="profile"><img src="assets/icons/user-profile-icon.svg" alt="">Profile</div>
+      <div id="more"><img src="assets/icons/more-options-ellipsis-icon.svg" alt="">More</div>
+      <div class="tweet-button">Tweet</div>
+      <div></div>
+      <div></div>
+    </section>
+
+    <section class="main">
+      <div>Home</div>
+      <div class="compose-grid">
+        <div class="profile-pic"></div>
+        <div class="compose-post">
+          <div class="enter-text" contenteditable>What's new?</div>
+          <div class="post-attachments">
+            <div><img src="assets/icons/picture-icon.svg" alt=""></div>
+            <div><img src="assets/icons/gif-file-icon.svg" alt=""></div>
+            <div><img src="assets/icons/poll-icon.svg" alt=""></div>
+            <div><img src="assets/icons/happy-icon.svg" alt=""></div>
+            <div><img src="assets/icons/information-date-calendar-icon.svg" alt=""></div>
+            <div></div>
+            <div class="tweet-button">Tweet</div>
+          </div>
+        </div>
+      </div>
+      <div class="feed">
+        <div class="feed-post">
+          <div class="feed-profile-pic"></div>
+          <div class="feed-body">
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
+        </div>
+        <div class="feed-post">
+          <div class="feed-profile-pic"></div>
+          <div class="feed-body">
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
+        </div>
+        <div class="feed-post">
+          <div class="feed-profile-pic"></div>
+          <div class="feed-body">
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
+        </div>
+        <div class="feed-post">
+          <div class="feed-profile-pic"></div>
+          <div class="feed-body">
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
+        </div>
+        <div class="feed-post">
+          <div class="feed-profile-pic"></div>
+          <div class="feed-body">
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
+        </div>
+        <div class="feed-post">
+          <div class="feed-profile-pic"></div>
+          <div class="feed-body">
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
+        </div>
+        <div class="feed-post">
+          <div class="feed-profile-pic"></div>
+          <div class="feed-body">
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
+        </div>
+        <hr>
+      </div>
+
+    </section>
+
+    <section class="happening"> 
+      <div>Search</div>
+      <div>What's happening<hr></div>
+      <div>Who to follow<hr></div>
+
```

Comment:
There are a lot of empty divs's

Reply:
True, I suppose I could've used p tags here instead